src/test/java/org/broadinstitute/barclay/argparser/CommandLineArgumentParserTest.java

```diff
@@ -534,6 +534,22 @@ public void testCollectionThatCannotBeAutoInitialized() {
 
     }
 
+    // Should be able to use null to reset a list (with no ranges specified) to empty
+    @Test
+    public void testResetListToNull() {
+        class ResetListToNull {
+            @Argument
+            public List<String> LIST = makeList("foo", "bar");
```

Comment:
I will use instead a numeric list to check that this is correct. That was one of my bugs (see #41) and maybe it is better to be sure that it is not broken also for collections.